director/authentication/json_web_token.py

```diff
@@ -0,0 +1,31 @@
+from flask import jsonify, request
+import jwt 
+import datetime
+from functools import wraps
+secret = 'pass'
+
+"""
+THIS function requires the use of a json web token in order to 
+be accepted 
+"""
+def token_required(f):
```

Comment:
This method should also be somewhere in the `api` module.

Reply:
should i move the hole file ?